mango-core/src/main/scala/org/bdgenomics/mango/layout/AlignmentRecordLayout.scala

```diff
@@ -121,37 +110,24 @@ object MergedAlignmentRecordLayout extends Logging {
 }
 
 /**
- * An extension of TrackedLayout for AlignmentRecord data
+ * AlignmentRecord data
  *
- * @param values The set of (Reference, AlignmentRecord) tuples to lay out in tracks
+ * @param values The set of (Reference, AlignmentRecord) tuples
  */
-class AlignmentRecordLayout(values: Iterator[(ReferenceRegion, CalculatedAlignmentRecord)]) extends TrackedLayout[CalculatedAlignmentRecord, ReadsTrackBuffer] with Logging {
-  val sequence = values.toList
-  var trackBuilder = new ListBuffer[ReadsTrackBuffer]()
 
-  val readPairs: Map[String, List[(ReferenceRegion, CalculatedAlignmentRecord)]] = sequence.groupBy(_._2.record.getReadName)
-  addTracks
-  trackBuilder = trackBuilder.filter(_.records.nonEmpty)
+class AlignmentRecordLayout(values: Iterator[(ReferenceRegion, CalculatedAlignmentRecord)]) with Logging {
+  val sequence = values.toList
 
-  def addTracks {
-    readPairs.foreach {
+  def collect: Iterator[CalculatedAlignmentRecord] = {
+    val records = new ListBuffer[CalculatedAlignmentRecord]
```

Comment:
Is that what this line of code does?
`sequence.groupBy(_._2.record.getReadName)`

Does that mean that I should be returning tuples of [String, CalculatedAlignmentRecord]?